src/timepicker/TimePickerContent.ts

```diff
@@ -21,6 +24,40 @@ export const useTimePickerContent = createHook<
   name: "TimePickerContent",
   compose: usePickerBaseContent,
   keys: TIME_PICKER_CONTENT_KEYS,
+
+  useProps(_, { ref: htmlRef, onKeyDown: htmlOnKeyDown, ...htmlProps }) {
+    const ref = React.useRef<HTMLElement>(null);
+
+    const onKeyDown = React.useMemo(() => {
+      return createOnKeyDown({
+        onKeyDown: htmlOnKeyDown,
+        keyMap: () => {
+          return {
+            ArrowRight: () => {
+              if (ref.current) {
```

Comment:
I think we can put the `if (ref.current)` check outside ArrowRight & ArrowLeft. before the return

Reply:
Throws TS null assertion error.

Reply:
Added Guard Clause instead as discussed.